### DISCUSSION TITLE
How to run bibtex?

### DISCUSSION THREAD

**Discussion by netw0rkf10w** [James-Yu/LaTeX-Workshop#4081] [Q&A]
Hello,

I have added a "bibtex" recipe (with a single `bibtex` command). This recipe appears in the list of commands:

<img width="278" alt="Screenshot 2023-12-03 at 12 49 43" src="https://github.com/James-Yu/LaTeX-Workshop/assets/8569515/8ee7c145-76f9-4521-a987-323c497b72e9">

However, when I click on it, nothing happens. Could you please help?

Thank you very much!

**Comment by James-Yu** [comment]
Not an extension bug.
